addons/one-click-fluffyscratch-login/notes.txt

```diff
@@ -0,0 +1 @@
+If the project ID changes, update it in consts.json (projectId) and addon.json (userscripts[0].matches, userstyles[0].matches)
```

Comment:
I feel like this should be a comment in userscript.js but whatever

Reply:
ok